lib/msf/base/sessions/command_shell.rb

```diff
@@ -107,13 +107,11 @@ def bootstrap(datastore = {}, handler = nil)
           banner.gsub!(/[^[:print:][:space:]]+/n, "_")
           banner.strip!
 
-          banner = %Q{
+          session_info = @banner = %Q{
```

Comment:
Keep the session info and banner separate. This allows the SSH sessions to have both, but set the session_info to the more consistent value of `SSH user @ peerhost:port` which is similar to Meterpreter.